assets/_scss/_pagenation.scss

```
@@ -19,6 +19,18 @@
 		}
 	}
 
+	&-numbers {
+		+ .wp-block-query-pagination-next {
+			margin-inline-start: .5em;
+		}
+	}
+
+	&-previous {
+		+ .wp-block-query-pagination-numbers {
+			margin-inline-start: .5em;
+		}
+	}
+
```

@sysbird ありがとうございまっす。
ここ + なしで普通に下記のような感じでいいかなと思うのですがどうでしょう（・ｗ・？

```suggestion
	.wp-block-query-pagination-previous {
		margin-inline-end: .5em;
	}

	.wp-block-query-pagination-next {
		margin-inline-start: .5em;
	}
```

---

@sysbird あ、いや、コメントのページナビゲーションも関係するからこれじゃダメかも。
コメントのページ送りでも問題なく表示されるか確認の上、必要に応じて調整よろしくお願いいたします。

---

> ここ + なしで普通に下記のような感じでいいかなと思うのですがどうでしょう

@kurudrive 
あら、これでいけますね
コアの CSS に負けてうまくいかないように見えたのですが、気のせいでした
ありがとうございます！